tests/unit/s2n_mem_usage_test.c

```diff
@@ -22,6 +22,12 @@
     #include <sys/sysctl.h>
     /* clang-format on */
     #include <sys/user.h>
+#elif defined(__OpenBSD__)
+    #undef _POSIX_C_SOURCE
+    #include <kvm.h>
+    #include <unistd.h>
+    #include <sys/types.h>
+    #include <sys/sysctl.h>
```

Comment:
I think clang format is complaining about the indented include.
```suggestion
#elif defined(__OpenBSD__)
    #undef _POSIX_C_SOURCE
    /* clang-format off */
    #include <kvm.h>
    #include <unistd.h>
    #include <sys/types.h>
    #include <sys/sysctl.h>
    /* clang-format on */
```

Reply:
Thanks for that. I was having a hard time understanding what clang-format was complaining about.